lyncs_io/testing.py

```diff
@@ -60,11 +60,40 @@
     ],
 )
 
+# includes all dtypes as dtype_loop but float16 as it's not supported
+dtype_mpi_loop = mark.parametrize(
+    "dtype_mpi",
```

Comment:
Still name the variable `dtype` so it is not needed to change the signature of all test functions